instance/models/mixins/openedx_config.py

```diff
@@ -83,19 +83,7 @@ def _get_configuration_variables(self):
                     # use different contact page
                     "CONTACT": "/contact",
                 },
-                # default values
-                "MKTG_URL_LINK_MAP": {
```

Comment:
Completly removed from `EDXAPP_LMS_ENV_EXTRA`. 

MKTG_URL_LINK_MAP - will contain a list of only disabled pages.